src/libraries/System.Security.Cryptography.Cng/tests/PropertyTests.cs

```diff
@@ -44,27 +31,15 @@ public static void CreatePersisted_PlatformEccKeyHasKeySize(string algorithm, in
         [OuterLoop("Hardware backed key generation takes several seconds.")]
         public static void CreatePersisted_PlatformRsaKeyHasKeySize(int keySize)
         {
-            CngKey key = null;
+            CngProperty keyLengthProperty = new CngProperty("Length", BitConverter.GetBytes(keySize), CngPropertyOptions.None);
+            CngPlatformProviderKey platformKey = new CngPlatformProviderKey(
```

Comment:
nit: to avoid the extra using block below we could just add the `using` keyword here

```suggestion
            using CngPlatformProviderKey platformKey = new CngPlatformProviderKey(
```

Reply:
This is another one that we try to avoid. It's "okay" in tests I think but we just don't use it at all in System.Security.

For example: https://github.com/dotnet/runtime/pull/59674#discussion_r718025503

Reply:
Yeah, @bartonjs hates the blockless using.  It may not be used in System.Security code, because dispose scopes should be VERY clear.